server/dash/dashboards/neighborhood_recent.py

```diff
@@ -186,7 +171,12 @@ def update_text(selected_council):
 def update_figure(selected_council):
     figure_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['createdDate', 'typeName'])['srnumber'].count().reset_index()  # noqa
     figure_df.typeName = figure_df.typeName.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))  # noqa
-
+    if figure_df.shape[0] == 0:
+        figure_df = pd.DataFrame(columns=['createdDate', "srnumber", "typeName"])
```

Comment:
Note: I know that the single quote vs double quote usage is all over the place in this file, but in the future, we should try to stick to one or the other in our codebase. I prefer double quotes.

Reply:
Updated that. Thanks for the catch!